ToolTipMenu.podspec

```diff
@@ -0,0 +1,22 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name         = "ToolTipMenu"
+  s.version      = package['version']
+  s.summary      = package['description']
+  s.license      = package['license']
+
+  s.authors      = package['author']
+  s.homepage     = package['repository']['url']
+  s.platform     = :ios, "10.0"
+  s.ios.deployment_target = '10.0'
+  s.tvos.deployment_target = '10.0'
```

Comment:
Thanks for this PR and your fork! Needed on RN 0.60 as it's requiring pods as default, and packages not using pods run into `'React/RCTViewManager.h' file not found` [issues](https://stackoverflow.com/questions/46067422/react-rctviewmanager-h-file-not-found).

However, your ios target version of 10.0 is higher than need be. On running `pod update`, I get this error:
```
[!] CocoaPods could not find compatible versions for pod "ToolTipMenu":
  In Podfile:
    ToolTipMenu (from `../node_modules/react-native-tooltip`)

Specs satisfying the `ToolTipMenu (from `../node_modules/react-native-tooltip`)` dependency were found, but they required a higher minimum deployment target.
```

I `patch-package`'ed your fork to delete the above 2 lines with deployment_target, and changed s.platform to 9.0. Like https://github.com/react-native-community/react-native-webview/blob/master/react-native-webview.podspec as reference podspec. Then `pod update` runs fine.